trikot-bluetooth/swift-extensions/TrikotAttributeProfileCharacteristic.swift

```diff
@@ -49,4 +49,3 @@ class TrikotAttributeProfileCharacteristic: NSObject, AttributeProfileCharacteri
         super.init()        
     }
 }
-
```

Comment:
At line 7, there is a `let event = frozenSubject()` that needs to be replaced with: `let event: Publisher = Publishers().publishSubject()`